### DISCUSSION TITLE
Copy email to clipboard after creation

### DISCUSSION THREAD

**Discussion by titoesteves** [simple-login/app#514] [Feature Requests]
Feature request to copy the newly created email to the clipboard after creating it. The current behavior requires an additional step(clicking the email) after the page load in order to copy the alias to the clipboard. 

Steps to reproduce:
1. Create new email alias

After creating the new email alias, we need to click on the newly created email in order to copy to clipboard. Would be nice if it is already copied to the clipboard after creation.

**Comment by WhamboMPS** [comment]
Yes, yes, yes; +1, +1, +1. 🥇
